fluid_dynamics/use_cases/kelvin_helmholtz_instability/source/ProjectParameters.json

```diff
@@ -0,0 +1,173 @@
+{
+    "problem_data"                     : {
+        "problem_name"    : "setup",
+        "parallel_type"   : "OpenMP",
+        "echo_level"      : 0,
+        "start_time"      : 0.0,
+        "end_time"        : 10.0
+    },
+    "solver_settings"                  : {
+        "model_part_name"              : "MainModelPart",
+        "domain_size"                  : 2,
+        "solver_type"                  : "Monolithic",
+        "model_import_settings"        : {
+            "input_type"     : "mdpa",
+            "input_filename" : "kelvin_helmholtz"
+        },
+        "echo_level"                  : 1,
+        "consider_periodic_conditions": true,
```

Comment:
this does not yet use the B&S you wrote I think
Or does it?

Reply:
In FluidDynamicsApplication ... no not yet .. that is why I mentioned in the very beginning that this PR should wait for the other ones !

Reply:
but again the changes required to use the B&S in FluidDynamicsApp are not very big. I have them already on local branch and will make a PR once the other PR is through !

Reply:
ok :+1: